### PR TITLE
feat: Implement SentryModule and PureConfig integration

### PR DESCRIPTION
**Pull request by jakubjanecek**

### CODE REVIEW

File: http4s-server-blaze/src/main/scala/com/avast/sst/http4s/server/Http4sBlazeServerModule.scala [hunk 1]
```diff
@@ -9,6 +9,7 @@ import org.http4s.server.blaze.BlazeServerBuilder
 
 import scala.concurrent.ExecutionContext
 import scala.concurrent.duration.Duration
+
```
Comment (hanny24) [inline http4s-server-blaze/src/main/scala/com/avast/sst/http4s/server/Http4sBlazeServerModule.scala]: why?

Comment (jakubjanecek) [reply http4s-server-blaze/src/main/scala/com/avast/sst/http4s/server/Http4sBlazeServerModule.scala]: There was a missing newline.

File: sentry-pureconfig/src/main/scala/com/avast/sst/sentry/pureconfig/implicits.scala [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import pureconfig.generic.ProductHint
+
+/** Contains [[pureconfig.ConfigReader]] instances with default "kebab-case" naming convention. */
+object implicits extends ConfigReaders {
+
+  /** Contains [[pureconfig.ConfigReader]] instances with "kebab-case" naming convention.
+    *
+    * This is alias for the default `implicits._` import.
+    */
+  object KebabCase extends ConfigReaders
```
Comment (augi) [inline sentry-pureconfig/src/main/scala/com/avast/sst/sentry/pureconfig/implicits.scala]: I don't see any usage of `KebabCase` - is it ok?

Comment (jakubjanecek) [reply sentry-pureconfig/src/main/scala/com/avast/sst/sentry/pureconfig/implicits.scala]: Yes, this is for the users if they want to use that naming convention, it's not used in SST.

**Pull request merged**